spec/app/domain/authentication/authn-oidc/update_input_with_username_from_id_token_spec.rb

```diff
@@ -247,6 +262,108 @@ def request_body(request)
           expect { subject }.to raise_error(::Errors::Authentication::RequestBody::MissingRequestParam)
         end
       end
+
+      context "with valid id token in header" do
+        subject do
+          input_ = Authentication::AuthenticatorInput.new(
+            authenticator_name: 'authn-oidc',
+            service_id: 'my-service',
+            account: 'my-acct',
+            username: nil,
+            credentials: request_body(authenticate_id_token_request),
+            client_ip: '127.0.0.1',
+            request: authenticate_id_token_request
+          )
+
+          ::Authentication::AuthnOidc::UpdateInputWithUsernameFromIdToken.new(
+            verify_and_decode_token: authenticate_id_token_request_id_token_in_header_field,
+            validate_account_exists: mock_validate_account_exists(validation_succeeded: true)
+          ).call(
+            authenticator_input: input_
+          )
+        end
+      end
+
+      context "with valid id token in header" do
```

Comment:
This context has the exact name as above. Can you update it to describe its true purpose?

Reply:
deleted it